lib/index.ts

```diff
@@ -0,0 +1,63 @@
+import * as _ from 'lodash';
+import { parse } from './indent-parser';
+
+export { parseLockFile };
+
+export interface PkgNode {
+  name: string;
+  version: string;
+  restriction: string;
+  // ignoring for now - addressing later
+  // depType?: DepType;
+  // targetFrameworks?: string[];
+}
+
+export interface PkgTree {
+  name: string;
```

Comment:
`type: nuget`

Reply:
Could this go here https://github.com/snyk/snyk-nuget-plugin/blob/master/lib/index.js#L27 instead? We feel this might be the more appropriate place to inject that kind of info.

Reply:
Sure